doc/examples/uuid.rst

```diff
@@ -0,0 +1,290 @@
+one of these drivers was in use. As example, consider the following situation:
+
+* Application ``M`` written in C# generates a UUID and uses it as the ``_id``
+  of a document that it proceeds to insert into the ``uuid_test`` collection of
+  the ``example_db`` database. Let's assume that the canonical textual
+  representation of the generated UUID is::
+
+    00112233-4455-6677-8899-aabbccddeeff
+
+* Application ``N`` written in Python attempts to ``find`` the document
+  written by application ``M`` in the following manner::
+
+    from uuid import UUID
+    collection = client.example_db.uuid_test
+    results = collection.find({'_id': UUID('00112233-4455-6677-8899-aabbccddeeff')})
+
+  In this instance, the ``results`` cursor will never contain the document that
+  was inserted by application ``M`` in the previous step. This is because of
+  the different byte-order used by the C# driver for representing UUIDs as
+  BSON Binary. The following query, on the other hand, will successfully find
+  this document::
+
+    results = collection.find({'_id': UUID('33221100-5544-7766-8899-aabbccddeeff')})
+
+As this example demonstrates, differing byte-order can hamper
+interoperability between applications that use different drivers. To workaround
+this problem, users can configure their ``MongoClient`` with the appropriate
+:class:`~bson.binary.UuidRepresentation`.
+
+.. _configuring-uuid-representation:
+
+Configuring a UUID Representation
+---------------------------------
+
+Setting a :class:`~bson.binary.UuidRepresentation` configures
+PyMongo's behavior while encoding/decoding native UUID types to/from BSON.
+Applications can set the UUID representation in one of the following ways:
+
+#. Using the ``uuidRepresentation`` URI option::
+
+  client = MongoClient("mongodb://a:27107/?uuidRepresentation=javaLegacy")
+
+  Valid values are ``pythonLegacy``, ``javaLegacy``, ``csharpLegacy``,
+  ``standard`` and ``unspecified``.
+
+#. Using the ``uuid_representation`` kwarg option::
+
+  from bson.binary import UuidRepresentation
+  client = MongoClient(uuid_representation=UuidRepresentation.PYTHON_LEGACY)
+
+#. By supplying a suitable :class:`~bson.codec_options.CodecOptions` instance::
+
+  from bson.codec_options import CodecOptions
+  csharp_opts = CodecOptions(uuid_representation=UuidRepresentation.CSHARP_LEGACY)
+  csharp_database = client.get_database('csharp_db', codec_options=csharp_opts)
+  csharp_collection = client.testdb.get_collection('csharp_coll', codec_options=csharp_opts)
+
+
+We now detail the behavior and use-case for each support UUID representation.
+
+The ``PYTHON_LEGACY`` UUID Representation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. attention:: This uuid representation should be used when reading UUIDs
+   written to MongoDB by existing applications that use the Python driver
+   and don't explicitly set a UUID representation.
+
+.. attention:: As of PyMongo 3.11.0,
```

Comment:
This implies to me that this is new in 3.11.0. Better to say that PYTHON_LEGACY is the default and has been since PyMongo 2.9, the version it was introduced in.